.vscode/settings.json

```diff
@@ -1,5 +1,6 @@
 {
     "cmake.configureOnOpen": true,
+    "editor.renderWhitespace": "all",
```

Comment:
Do we really want to add this? Is it somehow related to basic formatting?

Reply:
I'm going to fix this configuration. You're right, It's not welcomed for everyone.